src/components/common/Sidebar/Sidebar.js

```diff
@@ -184,6 +182,8 @@ const Sidebar = props => {
       <Layout className="site-layout">
         <Content style={{ margin: '2vh 1vw' }}>
           <Content>
+            {/* This is where the component rendering magic happens. Depending on the value of "key" in Menu.Item, the render variable will change,
```

Comment:
Love the comments, this helps!

Reply:
perfectly descriptive for those that take the underdog devs project on later. Nicely done.